source/includes/actions/_index.md

```diff
@@ -0,0 +1,38 @@
+## Get All Actions
+
+> `GET http://localhost:3000/api/v6/meta/actions`
+
+> Example response
+
+```json
+[
+  {
+    "id": 1,
+    "name": "regenerate",
+    "label": "Regenerate",
+    "description": null,
+    "uuid": "91e8f48c-bee6-41d8-b451-73b94b8d6342",
+    "created_at": "2020-11-09T13:08:36.000+00:00",
+    "updated_at": "2020-11-09T13:08:36.000+00:00",
+    "client_id": null
```

Comment:
client_id can't and shouldn't be present in the response, nor should created_at or updated_at in some resources.
We use uuid to pull resources, so id shouldn't be visible either.

Reply:
@pacMakaveli in that case, we'll need serializers to specify what we return. 

do you agree? if so i will make a PR to EDi adding serializers

Reply:
That's fine with me. Not all information is relevant to 3rd parties.
However, we'll need a new API, v7. If we deploy and the app works fine with the new API, we can remove it and leave v6 default.